main.py

```diff
@@ -108,10 +108,13 @@ def forward_decoder(self, x):
         recon = self.decoder(x)
         return recon
 
-    def forward(self, x):
+    def forward(self, x, path=None):
         mu, log_var = self.forward_encoder(x)
         hidden = self.reparameterize(mu, log_var)
         recon = self.forward_decoder(hidden)
+        if path:
+            np.savez_compressed('D:/data/pix3d/'+path,
```

Comment:
Please kindly add configurable path instead of hard-coded one. Thus, it's better to add these config into `__init__`

Reply:
I will modify this in the next version.